monai/utils/export_utils.py

```diff
@@ -0,0 +1,138 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: LicenseRef-NvidiaProprietary
+#
+# NVIDIA CORPORATION, its affiliates and licensors retain all intellectual
+# property and proprietary rights in and to this material, related
+# documentation and any modifications thereto. Any use, reproduction,
+# disclosure or distribution of this material and related documentation
+# without an express license agreement from NVIDIA CORPORATION or
+# its affiliates is strictly prohibited.
+
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from typing import Callable
+
+import torch.nn as nn
+
+from .cast_utils import CastToFloat
+
+
+def simple_replace(base_t: type[nn.Module], dest_t: type[nn.Module]) -> Callable[[nn.Module], nn.Module | None]:
```

Comment:
May I know why putting all these utils here instead of:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/networks/utils.py?
I see all your functions are related to the network module, similar to:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/networks/utils.py#L1068
@yiheng-wang-nv @KumoLiu @ericspod What do you think?

Thanks.

Reply:
Well, you proposed two locations at first, so I picked /utils to avoid merging two implementations in the same .py. 
Now after some refactoring, I think I can reuse some of functions in networks/util.py and move trt wrapper there (or rather networks/trt_wrapper.py). Please let me know what you think!

Reply:
Great, thanks.